VERSIONING.md

```diff
@@ -57,16 +69,16 @@ The Serverless Framework supports the major cloud providers Node.js runtime vers
 
 1. Is it okay to mark a feature as deprecated in version 1.4.0 and then remove it in 1.8.0
 
-No, since this is a breaking change it should trigger a major version bump to 2.0.0
+Yes, a breaking change to individual features may happen with a MINOR release
 
 2. Can we change everything in a major version bump?
 
-Yes, this is the purpose of major version bumps. Ideally every breaking change has a clear and well documented migration path. In best case the features were already introduced earlier and upgrading is not a dealbreaker.
+Yes, a new major will most likely announce a completely new (or significantly changed) shape of the framework
```

Comment:
> a new MAJOR release will most likely introduce a completely new (or significantly changed) framework design

or `architecture` rather than `design`.